process_report/process_report.py

```diff
@@ -62,16 +64,42 @@ def is_old_pi(old_pi_dict, pi, invoice_month):
     return False
 
 
+def get_invoice_bucket():
+    try:
+        s3_resource = boto3.resource(
+            service_name="s3",
+            endpoint_url=os.environ.get(
+                "S3_ENDPOINT", "https://s3.us-east-005.backblazeb2.com"
+            ),
+            aws_access_key_id=os.environ["S3_KEY_ID"],
+            aws_secret_access_key=os.environ["S3_APP_KEY"],
+        )
+    except KeyError:
+        print("Error: Environment variables for S3 authentication not set")
```

Comment:
The user wouldn't know what the name of the environment variables they need to set by looking at this error message. You should include that. 
```python
print("Error: Please set the environment variables S3_KEY_ID and S3_APP_KEY")
```